service/service.go

```diff
@@ -56,8 +57,42 @@ type Service struct {
 	DeployedAt time.Time `hash:"-"`
 }
 
-// getDependency returns dependency dependencyKey or a not found error.
-func (s *Service) getDependency(dependencyKey string) (*Dependency, error) {
+// StatusType of the service.
+type StatusType uint
+
+// Possible statuses for service.
+const (
```

Comment:
I don't see any reason to keep this in the service and not put it in the manager but i could be ok to keep it like that

Reply:
why not, it's a generic status indicator for services. it's not specific to Docker Service. It'll be used by Kubernetes or in any other implementation of Manager.

Reply:
let's continue this conversation outside the PR: https://github.com/mesg-foundation/core/issues/958